### PR TITLE
DOCS-7431 Changes to Reflect Tokenization with Global Namespaces

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: modules/ROOT/pages/tokenization.adoc [hunk 1]
```diff
@@ -24,6 +24,23 @@ A connection is established between the API gateway and the tokenization service
 
 The API gateway replaces the credit card number with the tokenized data and the request is sent to the app. 
 
+
+== Prerequisites
+
+To configure and use the tokenization service, you must:
+
+* Have the Anypoint Security - Edge entitlement for your Anypoint Platform account. +
```
Comment (clies212) [inline modules/ROOT/pages/tokenization.adoc]: Should say -- meet Runtime Fabric Pre-Reqs

**Review by clies212 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/tokenization.adoc [hunk 2]
```diff
@@ -24,6 +24,23 @@ A connection is established between the API gateway and the tokenization service
...
+
+== Prerequisites
+
+To configure and use the tokenization service, you must:
+
+* Meet all the prerequisites and requirements for running Anypoint Runtime Fabric +
+If you don't see *Security* in *Management Center* or the *Tokenization Service* tab in Runtime Manager, contact your customer success manager to enable the tokenization service for your account. +
+If you do not see the *Tokenization Service* tab in Runtime Manager, contact your customer success manager to enable the tokenization service for your account. 
+* Have the correct permissions to manage tokenization. +
+See xref:grant-tokenization-permissions.adoc[Granting Tokenization Permissions]
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/tokenization.adoc]:
```suggestion
See xref:grant-tokenization-permissions.adoc[Granting Tokenization Permissions].
```

File: modules/ROOT/pages/tokenization.adoc [hunk 3]
```diff
@@ -24,6 +24,23 @@ A connection is established between the API gateway and the tokenization service
 
 The API gateway replaces the credit card number with the tokenized data and the request is sent to the app. 
 
+
+== Prerequisites
+
+To configure and use the tokenization service, you must:
+
+* Meet all the prerequisites and requirements for running Anypoint Runtime Fabric +
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/tokenization.adoc]:
```suggestion
* Meet all the prerequisites and requirements for running Anypoint Runtime Fabric. +
```

**Review by wwarner-salesforce — DISMISSED** [review]

File: modules/ROOT/pages/tokenization.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/tokenization.adoc]: FIxed

**Review by wwarner-salesforce — APPROVED** [review]

File: modules/ROOT/pages/tokenization.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/tokenization.adoc]: Revise to: Meet all the prerequisites and requirements for running Anypoint Runtime Fabric.

**Review by clies212 — APPROVED** [review]
Looks Good!

**Pull request merged**